src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -114,4 +131,30 @@ void notRemoveNonOrphans() {
         // then
         assertThat(authorService.count()).isOne();
     }
+    
+    @Test
+    void notEmptyFirstName() {
+    	// given
+    	Author noFirst = new Author(" ", "Gaarder");
```

Comment:
For readability, could you please rename this to `authorWithEmptyFirstName`? To me, no first name would be null (absence of a value)? Also,